source/solvers/heat_transfer.cc

```diff
@@ -906,7 +899,7 @@ HeatTransfer<dim>::set_initial_conditions()
                            newton_update);
   nonzero_constraints.distribute(newton_update);
   present_solution = newton_update;
-  finish_time_step();
+  percolate_time_vectors();
```

Comment:
Wait I don't understand something here. I though we were only percolating the time vectors manually from the Navier-Stokes solver here? But post-process percolates the time vector also? Does this not risk a double percolation?

Reply:
Yes, I wondered that myself! I am not sure why this is done so, but I like that having removed `finish_time_step` clarifies what is being done. Once I fix the tests I will try to remove that to see how it goes